pkg/build/netbsd.go

```diff
@@ -66,3 +74,59 @@ func (ctx netbsd) clean(kernelDir string) error {
 	// machine.
 	return nil
 }
+
+// Copy the compiled kernel to the qemu disk image using ssh
+func CopyKernelToDisk(outputDir string) error {
+	temp := []byte(`
+{
+	"snapshot": false,
+	"mem": 1024
+}	`)
+	VMconfig := (*json.RawMessage)(&temp)
+	// Create config for booting the disk image
+	cfg := &mgrconfig.Config{
+		Workdir:      outputDir,
+		Image:        filepath.Join(outputDir, "image"),
+		SSHKey:       filepath.Join(outputDir, "key"),
+		SSHUser:      "root",
+		TargetOS:     "netbsd",
+		TargetArch:   "amd64",
+		TargetVMArch: "amd64",
+		Type:         "qemu",
+		VM:           *VMconfig,
+	}
+	// Create a VM pool
+	pool, err := vm.Create(cfg, false)
+	if err != nil {
+		return fmt.Errorf("failed to create a VM Pool : %v", err)
+	}
+	// Create a new reporter instance
+	reporter, err := report.NewReporter(cfg)
+	if err != nil {
+		return fmt.Errorf("failed to create a Reporter : %v", err)
+	}
+	// Create a VM instance (We need only one)
+	inst, err := pool.Create(0)
+	if err != nil {
+		return fmt.Errorf("failed to create the VM Instance : %v", err)
+	}
+	defer inst.Close()
+	// Copy the kernel into the disk image and replace it
+	// This makes use of the fact that the file is copied by default to /
+	kernel, err := inst.Copy(filepath.Join(outputDir, "netbsd"))
+	if err != nil {
+		return fmt.Errorf("error Copying the kernel %v: %v", kernel, err)
+	}
+	// Run poweroff so that the copied image is stored properly
+	// Due to some reason poweroff command isn't recognized over ssh
+	// hence we use /sbin/poweroff.
+	outc, errc, err := inst.Run(time.Minute, nil, "/sbin/poweroff")
+	if err != nil {
+		return fmt.Errorf("error powering off the instance %v : %v", outc, errc)
+	}
+	rep := inst.MonitorExecution(outc, errc, reporter, vm.ExitNormal)
+	if rep != nil {
+		return fmt.Errorf("error executng poweroff : %v", rep)
```

Comment:
`rep` will print badly with `%v`, it's a struct with lots of fields:
https://github.com/google/syzkaller/blob/master/pkg/report/report.go#L31-L53

We can start with just printing rep.Title with a one-line crash title. rep.Output will contain whole kernel and command output, which can be quite large, so we can add it later if needed.